doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3435,6 +3435,13 @@ The general command to add a hint to some databases :n:`{+ @ident}` is
 
       Declares each :n:`@ident` as a transparent or opaque constant.
 
+   .. cmdv:: Hint %( Variables %| Constants %) %( Transparent %| Opaque %)}
+      :name: Hint ( Variables | Constants )
```

Comment:
The manual now supports giving the same description to several commands, here that would give:

```
.. cmdv:: Hint Variables %( Transparent %| Opaque %)
          Hints Constants %( Transparent %| Opaque %)
          :name: Hint Variables ( Transparent | Opaque ); Hint Constants ( Transparent | Opaque )
```
This would have the advantage of creating two separate index entries.

Reply:
We should do something similar to the above `Hint ( Transparent | Opaque )` definition BTW, then this would allow referencing one or the other with ```:cmd:`Hint Transparent` ``` and ```:cmd:`Hint Opaque` ```.

Reply:
Done

Reply:
Did you push?